packages/flutter/lib/src/widgets/scrollable.dart

```diff
@@ -50,13 +56,25 @@ abstract class Scrollable extends StatefulComponent {
            scrollDirection == Axis.horizontal);
   }
 
+  /// The scroll offset this widget should use when first created.
   final double initialScrollOffset;
+
+  /// The axis along which this widget should scroll.
   final Axis scrollDirection;
+
+  /// Called whenever this widget starts to scroll.
   final ScrollListener onScrollStart;
+
+  /// Called whenever this widget's scroll offset changes.
   final ScrollListener onScroll;
+
+  /// Called whenever this widget stops scrolling.
   final ScrollListener onScrollEnd;
+
+  /// Called to determine the offset to which scrolling should snap.
   final SnapOffsetCallback snapOffsetCallback;
-  final double snapAlignmentOffset;
+
+  final double snapAlignmentOffset; // What does this do?
```

Comment:
I looked into it and didn't quite understand what it was doing.  Hopefully @HansMuller can explain it to me and we can document it.